src/api-reference/cash-advance/v4.cash-advance.md

```diff
@@ -74,15 +70,9 @@ Creates a cash advance.
 ##### Template
 
 ```shell
-https://{datacenterURI}/cashadvance/v4/companies/{companyID}/cashadvances
+https://{datacenterURI}/cashadvance/v4/cashadvances
 ```
 
-##### Parameters
-
-Name|Type|Format|Description
```

Comment:
Hi @gaganpsgithub. Could you please keep this section and just place None. below it?

Thank you.